php/class-fieldmanager-plaintext.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Class file for Fieldmanager_Plaintext
+ *
+ * @package Fieldmanager
+ */
+
+/**
+ * Content field which renders escaped plaintext.
+ */
+class Fieldmanager_Plaintext extends Fieldmanager_Content {
+
+	/**
+	 * Override field_class.
+	 *
+	 * @var string
+	 */
+	public $field_class = 'plaintext';
+
+	/**
+	 * Render content.
+	 *
+	 * @param mixed $value Unused value.
+	 * @return string Rendered content.
+	 */
+	public function render_content( $content = '' ) {
+		return esc_html( $content );
```

Comment:
Late-breaking thought: Should we run this through `wpautop()` to ensure that it's not text sitting in a `<div>`?

Reply:
I don't have strong feelings on this either way. Are you throwing the idea out there? Or do you think we should go ahead and do it?

Reply:
My only goal is to ensure the content is wrapped in something semantic. Beyond that, I'm not picky. Without checking out the branch at this moment, if the content is currently within a `div` and using `wpautop` will make it not be in a `div`, then I'd take it. Maybe wrapping the return value in `<p>` and `</p>` could also do it.

Reply:
I had a chance to check out the branch and see that the content currently displays directly within `div.fm-item.fm-plaintext`. It seems reasonable to me to assume that a paragraph will accurately describe the kind of content that would be included with the plaintext field (with complex content going into the Markdown or HTML fields). For that reason, running the output through `wpautop()` would be my recommendation.

Reply:
I just pushed up a commit introducing `wpautop` to the plaintext field.